Source/SIMPLib/FilterParameters/DataArraySelectionFilterParameter.cpp

```diff
@@ -93,7 +88,7 @@ DataArraySelectionFilterParameter::RequirementType DataArraySelectionFilterParam
                                                                                                                 size_t allowedCompDim,
                                                                                                                 AttributeMatrix::Category attributeMatrixCategory)
 {
-  typedef QVector<size_t> QVectorOfSizeType;
+  using QVectorOfSizeType = std::vector<size_t>;
```

Comment:
This is a misleading and bloated type name.  If the name can be changed without breaking all the FilterParameters, consider using SizeVecType instead.
```suggestion
  using SizeVecType = std::vector<size_t>;
```

Reply:
I only found this alias in .cpp files, so it is unlikely that a mass search and replace for using and typedef versions will break anything.